requirements.txt

```diff
@@ -1,8 +1,8 @@
 mkdocs==1.0.4
 mkdocs-material==4.1.1
```

Comment:
In testing, it should be safe to update both `mkdocs` to its latest, and `mkdocs-material` to a version lower than 6.2.0.

This will also require some configuration change, however.

Reply:
I've updated in https://github.com/srobo/runbook/pull/124/commits/85d2a0aec5f0d5099fd3f2e5b9474c174fbe9d50, though not quite that far. From what I can tell we don't need config changes with the updates I've got here.